src/_data/sidenav.yml

```diff
@@ -60,6 +60,16 @@
 
 - title: App solutions
   children:
+    - title: AI
+      children:
+      - title: Overview
+        permalink: /resources/ai-overview
+      - title: generative_ai package
```

Comment:
From a branding perspective Google AI Dart SDK == google_generative_ai pkg. Let's try to keep it consistent!